plugins/woocommerce-admin/client/customize-store/index.tsx

```diff
@@ -121,7 +123,12 @@ export const customizeStoreStateMachineDefinition = createMachine( {
 	context: {
 		intro: {
 			hasErrors: false,
-			themeCards: [] as ThemeCard[],
+			themeData: {
+				themes: [] as ThemeCard[],
+				_links: {
+					browse_all: '',
```

Comment:
How about we set the default value to `/wp-admin/themes.php`? Then, we can remove the `?` from type [here](https://github.com/woocommerce/woocommerce/pull/40552/files#diff-80d98a03c5872cccb709f4d263fc91db352907c5e1283cc40e54ca0fb16c30feR29 ) and remove the check [here](https://github.com/woocommerce/woocommerce/pull/40552/files#diff-48c61607b03816c346b4ca5853ce88cbdc0b471c9deb58406ec45ff26bfe270aR73)

Reply:
> /wp-admin/themes.php

It would be good to use `getAdminLink` in case the site has moved the `wp-admin` folder for security.

Something like:

```
import { getAdminLink } from '@woocommerce/settings';
getAdminLink( 'themes.php' )
```

Reply:
Good suggestions! Fixed in https://github.com/woocommerce/woocommerce/pull/40552/commits/a191ca3a2cd9c62e047aa8782287275226440812

Reply:
@chihsuan @adrianduffell

Reply:
LGTM thanks @moon0326 !